lib/lightning_web/live/run_live/components.ex

```diff
@@ -100,21 +30,24 @@ defmodule LightningWeb.RunLive.Components do
     assigns = assigns |> assign_new(:status, fn -> nil end)
 
     ~H"""
-    <tr class="my-4 grid grid-cols-4 gap-4 rounded-lg bg-white">
+    <tr class="my-4 grid grid-cols-5 gap-4 rounded-lg bg-white">
       <th
         scope="row"
         class="my-auto whitespace-nowrap p-6 font-medium text-gray-900 dark:text-white"
       >
-        workFlowName
+        <%= @workflow.name %>
       </th>
+      <td class="my-auto p-6"><%= @reason.dataclip_id %></td>
       <td class="my-auto p-6">12i78iy</td>
-      <td class="my-auto p-6">UTC 24:20:60</td>
+      <td class="my-auto p-6">
+        <%= @last_attempt.last_run.finished_at |> Calendar.strftime("%c") %>
```

Comment:
I managed to get the app to throw an exception with runs that haven't finished. I've fixed the issue in the WorkOrder component, but not here.

We need a test for this.

Also the name of this function component is `work_order`, should it not be attempt or something else? I'm confused by the `Components.WorkOrder` and this?

Is this file being used anymore? What about `AttemptComponent`? Please remove this file is we're not using it.

Reply:
Only pill function are needed in this file. Removed work_order function